computer-und-recht.csl

```diff
@@ -5,7 +5,7 @@
     <title-short>CR</title-short>
     <id>http://www.zotero.org/styles/computer-und-recht</id>
     <link href="http://www.zotero.org/styles/computer-und-recht" rel="self"/>
-    <link href="https://www.zotero.org/styles/neue-juristische-wochenschrift" rel="template"/>
+    <link href="http://www.zotero.org/styles/neue-juristische-wochenschrift" rel="template"/>
```

Comment:
Yes, @dstillman correctly mentioned that this isn't merely an implementation issue, since the "independent-parent" link is used both as a dereferenceable URI and as an identifier, so it's probably better to stick to HTTP for now.

I'd like to enforce HTTP for the "template" link as well for consistency for the time being.